### PR TITLE
Add policies field to Persons

### LINKED ISSUE #8: As an insurance agent, I can link my client's contact to his insurance portfolio with a portfolio ID...
so that I can access them easily.

### LINKED ISSUE #17: Add policy ID field

### LINKED ISSUE #18: Modify add to be able to insert policy.

### LINKED ISSUE #19: Modify edit to take in policy ID parameter

### PR DESCRIPTION
**Pull request by swayongshen**
- Add policies field to Persons to link InsurancePolicies to a client.
- Change AddCommand and EditCommand to include InsurancePolicy.
- Fix tests to accommodate new changes.

### CODE REVIEW

**Review by NBH99 — COMMENTED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -24,12 +25,14 @@
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
             + PREFIX_ADDRESS + "ADDRESS "
+            + "[" + PREFIX_INSURANCE_POLICY + "Policy ID] "
```
Comment (jlxw48) [inline src/main/java/seedu/address/logic/commands/AddCommand.java]: maybe block letters? follow the format of the other fields

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 2]
```diff
@@ -41,6 +43,7 @@
             + "[" + PREFIX_PHONE + "PHONE] "
             + "[" + PREFIX_EMAIL + "EMAIL] "
             + "[" + PREFIX_ADDRESS + "ADDRESS] "
+            + "[" + PREFIX_INSURANCE_POLICY + "Policy Id] "
```
Comment (jlxw48) [inline src/main/java/seedu/address/logic/commands/EditCommand.java]: maybe block letters? follow the format of the other fields

File: src/main/java/seedu/address/logic/parser/ArgumentMultimap.java [hunk 3]
```diff
@@ -6,6 +6,7 @@
 import java.util.Map;
 import java.util.Optional;
 
+
```
Comment (jlxw48) [inline src/main/java/seedu/address/logic/parser/ArgumentMultimap.java:9]: can consider removing the extra blank line?

File: src/main/java/seedu/address/storage/JsonAdaptedInsurancePolicy.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    /**
+     * Constructs a {@code JsonAdaptedInsurancePolicy} with the given {@code policyId}.
+     */
+    @JsonCreator
+    public JsonAdaptedInsurancePolicy(String policyId) {
+        this.policyId = policyId;
+    }
+
+    /**
+     * Converts a given {@code Tag} into this class for Jackson use.
```
Comment (jlxw48) [inline src/main/java/seedu/address/storage/JsonAdaptedInsurancePolicy.java]: I think this JavaDoc comment can be further refined.

**Review by jlxw48 — CHANGES_REQUESTED** [review]
Just a few minor issues.

File: src/main/java/seedu/address/logic/commands/AddCommand.java (hunk 1 above)
Comment (swayongshen) [reply src/main/java/seedu/address/logic/commands/AddCommand.java]: Done

File: src/main/java/seedu/address/storage/JsonAdaptedInsurancePolicy.java (hunk 4 above)
Comment (swayongshen) [reply src/main/java/seedu/address/storage/JsonAdaptedInsurancePolicy.java]: Have changed Tag into InsurancePolicy @ line 24. Will be keeping the JavaDoc comment to follow the JsonAdaptedTag class from which I adapted the code.

**Comment by swayongshen** [comment]
Thanks for the review. Will be merging it

**Pull request merged**